src/test/java/org/junitpioneer/jupiter/ReportEntryExtensionTest.java

```diff
@@ -13,20 +13,28 @@
 import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junitpioneer.jupiter.ReportEntry.PublishCondition.*;
```

Comment:
because noboby currently runs the checkstyle target for test, we only run checkstyleMain (like it used to be) but we would need to run also checkstyleTest (or something like that, not 100% sure about the naming

Reply:
```
Task :checkstyleTest FAILED
[ant:checkstyle] /home/sschrottner/projects/github/junit-pioneer/src/test/java/org/junit/jupiter/engine/AbstractJupiterTestEngineTests.java:0: error: Missing package-info.java file.
[ant:checkstyle] /home/sschrottner/projects/github/junit-pioneer/src/test/java/org/junit/platform/engine/test/TestDescriptorStub.java:0: error: Missing package-info.java file.
[ant:checkstyle] /home/sschrottner/projects/github/junit-pioneer/src/test/java/org/junit/platform/engine/test/event/ExecutionEvent.java:0: error: Missing package-info.java file.
[ant:checkstyle] /home/sschrottner/projects/github/junit-pioneer/src/test/java/org/junitpioneer/AbstractPioneerTestEngineTests.java:0: error: Missing package-info.java file.
[ant:checkstyle] /home/sschrottner/projects/github/junit-pioneer/src/test/java/org/junitpioneer/jupiter/DefaultLocaleTests.java:0: error: Missing package-info.java file.
[ant:checkstyle] /home/sschrottner/projects/github/junit-pioneer/src/test/java/org/junitpioneer/jupiter/params/RangeSourceProviderTests.java:0: error: Missing package-info.java file.
[ant:checkstyle] /home/sschrottner/projects/github/junit-pioneer/src/test/java/org/junitpioneer/vintage/TestIntegrationTests.java:0: error: Missing package-info.java file.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':checkstyleTest'.
> Checkstyle rule violations were found. See the report at: file:///home/sschrottner/projects/github/junit-pioneer/build/reports/checkstyle/test.html
  Checkstyle files with violations: 7
  Checkstyle violations by severity: [error:7]
```

Reply:
Quoting @nicolaiparlog here:

> [14:18] nipafx: At first glance it seems that most of those should apply to tests as well, assuming there aren't any defaults hidden somewhere. We should discuss that on the issue.